test/framework/robot.py

```diff
@@ -567,6 +567,14 @@ def test_get_toolchain_hierarchy(self):
             {'name': 'gompi', 'version': '1.4.10'},
```

Comment:
Agreed, my mistake

On 18 Feb 2016 21:06, "Kenneth Hoste" <notifications@github.com<mailto:notifications@github.com>> wrote:

In test/framework/robot.py<https://github.com/hpcugent/easybuild-framework/pull/1619#discussion_r53374477>:

> @@ -567,6 +567,14 @@ def test_get_toolchain_hierarchy(self):
>              {'name': 'gompi', 'version': '1.4.10'},


I don't think gompi is a broken choice, because we always include the toolchain too, not only the deps.

So, dummy is considered as a subtoolchain because GCC is built with dummy

—
Reply to this email directly or view it on GitHub<https://github.com/hpcugent/easybuild-framework/pull/1619/files#r53374477>.


------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------
Forschungszentrum Juelich GmbH
52425 Juelich
Sitz der Gesellschaft: Juelich
Eingetragen im Handelsregister des Amtsgerichts Dueren Nr. HR B 3498
Vorsitzender des Aufsichtsrats: MinDir Dr. Karl Eugen Huthmacher
Geschaeftsfuehrung: Prof. Dr.-Ing. Wolfgang Marquardt (Vorsitzender),
Karsten Beneke (stellv. Vorsitzender), Prof. Dr.-Ing. Harald Bolt,
Prof. Dr. Sebastian M. Schmidt
------------------------------------------------------------------------------------------------
------------------------------------------------------------------------------------------------